gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -900,7 +899,7 @@ protected void configureSchemaAnnotationValidation(Project project,
                   task.setResolverPath(getDataModelConfig(project, sourceSet)); // same resolver path as generateDataTemplatesTask
                   task.setClassPath(project.getConfigurations() .getByName(SCHEMA_ANNOTATION_HANDLER_CONFIGURATION)
                                            .plus(project.getConfigurations().getByName(PEGASUS_PLUGIN_CONFIGURATION))
-                                           .plus(project.getConfigurations().getByName(JavaPlatformPlugin.RUNTIME_CONFIGURATION_NAME)));
+                                           .plus(project.getConfigurations().getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME)));
```

Comment:
Hi Teddy, have a question, this from behind the scene change to use "runtime" to "runtimeClasspath" right?
Are they getting the same configuration under the hood?
I just to know if it will change anything.

Reply:
In theory, it is not exactly the same. The `runtimeClasspath` configuration extends from the `runtime` configuration, but it also does from the newer `runtimeOnly` and `implementation` configurations. This is documented [here](https://docs.gradle.org/6.8.3/userguide/java_plugin.html#tab:configurations).

In practice, I don't expect it to change anything because very few people internally use `runtimeOnly` and `implementation`, and because `runtimeClasspath` is strictly larger than `runtime`, and enlarging the classpath in my experience has a lower chance of leading to an issue that shrinking the classpath.